src/lua/luaTask.c

```diff
@@ -227,9 +275,16 @@ static void lua_task(void *params)
         };
 
         int consecutive_failures = 0;
-        for(portTickType xLastWakeTime;;
-            vTaskDelayUntil(&xLastWakeTime, state.callback_interval)) {
-                xLastWakeTime = xTaskGetTickCount();
+	portTickType wake_tick = 0;
+        for(;;) {
```

Comment:
Testing it directly it seems the command is processing immediately, even if tickRate() is 1Hz, which seems good, but it seems there's something else I'm not understanding out of reviewing the code.